src/server/server_family.cc

```diff
@@ -161,6 +168,38 @@ std::string AbslUnparseFlag(const ReplicaOfFlag& flag) {
   return (flag.has_value()) ? absl::StrCat(flag.host, ":", flag.port) : "";
 }
 
+bool AbslParseFlag(std::string_view in, CronExprFlag* flag, std::string* err) {
```

Comment:
if you are using absl flags framework now, then maybe we do not need to log the error ourselves since we return it via `err`. Have you tried passing invalid string? what do we log with this code?

Reply:
yes I've checked it, for example for only one argument like "2" we get next error "cron expression must have six fields"

Reply:
we can move this warning to ConfigRegistry::Set(...) function, where we take a pointer to this err

Reply:
I was not clear.
I think when passing `--snapshot_cron=foobar` as an argument dragonfly will print the error returned by this function. Therefore there is  no need to use `LOG(WARNING)` as well. Similarly, with "CONFIG SET" the error should be printed upon the command invocation. So again, I do not see why log warning is needed.